incubator/mongodb/README.md

```diff
@@ -0,0 +1,192 @@
+messy-hydra-mongodb-2 "primary" : "messy-hydra-mongodb-0.messy-hydra-mongodb.default.svc.cluster.local:27017",
+messy-hydra-mongodb-0 "primary" : "messy-hydra-mongodb-0.messy-hydra-mongodb.default.svc.cluster.local:27017",
+
+```
+
+Kill the primary and watch as a new master getting elected.
+```console
+$ kubectl delete pod <release-mongodb-0>
+
+pod "messy-hydra-mongodb-0" deleted
+```
+
+Delete all pods and let the petset controller bring it up.
+```console
+$ kubectl delete po -l app=mongodb
+$ kubectl get po --watch-only
+NAME                    READY     STATUS        RESTARTS   AGE
+messy-hydra-mongodb-0   0/1       Pending   0         0s
+messy-hydra-mongodb-0   0/1       Pending   0         0s
+messy-hydra-mongodb-0   0/1       Pending   0         7s
+messy-hydra-mongodb-0   0/1       Init:0/2   0         7s
+messy-hydra-mongodb-0   0/1       Init:1/2   0         27s
+messy-hydra-mongodb-0   0/1       Init:1/2   0         28s
+messy-hydra-mongodb-0   0/1       PodInitializing   0         31s
+messy-hydra-mongodb-0   0/1       Running   0         32s
+messy-hydra-mongodb-0   1/1       Running   0         37s
+messy-hydra-mongodb-1   0/1       Pending   0         0s
+messy-hydra-mongodb-1   0/1       Pending   0         0s
+messy-hydra-mongodb-1   0/1       Init:0/2   0         0s
+messy-hydra-mongodb-1   0/1       Init:1/2   0         20s
+messy-hydra-mongodb-1   0/1       Init:1/2   0         21s
+messy-hydra-mongodb-1   0/1       PodInitializing   0         24s
+messy-hydra-mongodb-1   0/1       Running   0         25s
+messy-hydra-mongodb-1   1/1       Running   0         30s
+messy-hydra-mongodb-2   0/1       Pending   0         0s
+messy-hydra-mongodb-2   0/1       Pending   0         0s
+messy-hydra-mongodb-2   0/1       Init:0/2   0         0s
+messy-hydra-mongodb-2   0/1       Init:1/2   0         21s
+messy-hydra-mongodb-2   0/1       Init:1/2   0         22s
+messy-hydra-mongodb-2   0/1       PodInitializing   0         25s
+messy-hydra-mongodb-2   0/1       Running   0         26s
+messy-hydra-mongodb-2   1/1       Running   0         30s
+
+
+...
+messy-hydra-mongodb-0 "primary" : "messy-hydra-mongodb-0.messy-hydra-mongodb.default.svc.cluster.local:27017",
+messy-hydra-mongodb-1 "primary" : "messy-hydra-mongodb-0.messy-hydra-mongodb.default.svc.cluster.local:27017",
+messy-hydra-mongodb-2 "primary" : "messy-hydra-mongodb-0.messy-hydra-mongodb.default.svc.cluster.local:27017",
+```
+
+Check the previously inserted key:
+```console
+$ kubectl exec $release-mongodb-1 -- /usr/bin/mongo --eval="rs.slaveOk(); db.test.find({key1:{\$exists:true}}).forEach(printjson)"
+
+MongoDB shell version: 3.2.8
+connecting to: test
+{ "_id" : ObjectId("57b180b1a7311d08f2bfb617"), "key1" : "value1" }
+```
+
+## Scaling
+
+Scaling should be managed by `helm upgrade`, which is the recommended way. 
+You can also scale up by modifying the number of replicas on the PetSet using `kubectl patch` or `kubectl apply`.
```

Comment:
Should we have a section on how to delete mongo, since PVC and pods do not auto delete with Pet Set?  The docs for Pet Set has a good example with the wait.